aisera/README.md

```diff
@@ -0,0 +1,95 @@
+   4. Enter the **Name**, **API Key** and **Application Key**.
+      ![my screenshot](images/integration_setup_3.png)
+   5. Click **OK**. 
+4. Create a data source for this integration.
+   1. Go to **Settings** > **Data Sources** > **New Data Source**.
+   2. In the search bar, select **Integration** in the previous step.
+   3. Enter a **Name** and **Schedule** to run the integration.
+   4. Select _Alerts Learning_ from **Functions**.
+      ![my screenshot](images/data_source_setup_1.png)
+      ![my screenshot](images/data_source_setup_2.png)
+5. Run the data source. You can also choose to run this data source based on a schedule.
+6. Once the alerts have been ingested, Aisera AIOps pipelines will run.
+7. AiOps pipelines will generate major incidents/clusters based on the category and severioty of incidents recieved.
+   1. Each major incident contains the following main components.
+      1. **Title**: Brief info about the major incident 
+      2. **Status**: Open/Close  
+      3. **Cluster**: Aisera generated cluster that this major incident belongs to. 
+      4. **Outage Probability**: Outage probability for this major incident.
+      5. **Impacted Business Services**
+      6. **Impacted Systems**: Infrastructure component impacted by this major incident.
+      7. **Impacted Storage**: Any DB/Storage that was impacted.
+      8. **Impacted Users**: Users which are being impacted by this major incident.
+      9. **Incidents**: Any Incident from ticketing systems that caused this major incident.
+      10. **Alerts**: Datadog alerts/logs that are part of this major incident.
+      11. **Change Request**: Any change request which caused this major incident.
+      12. **Timeline**: A timeline of the incident/alerts correlation
+      13. **Root Cause**: A root cause generated by the Aisera LLM model
+      14. **Causal Graphs**: Graph of business services
+      15. **Logs and Trace Analysis**
+      16. **Major Incident Connectivity Graph**: A visual representation of how users are impacted by specific alerts/services/infrastructure
+      17. **Score over Time**: AMI scores over time
+      18. Outage Probability over Time
+      19. **Major Incident Correlation Evidence**
+          ![my screenshot](images/aisera-major_incident_detail_1.png)
+          ![my screenshot](images/aisera-major_incident_detail_2.png)
+          ![my screenshot](images/aisera-major_incident_detail_3.png)
+          ![my screenshot](images/aisera-major_incident_detail_4.png)
+8. Once Aisera detects a major incident:
+- It notifies the designated persona on the channel of their choice.
+- Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the
+10. We can also put Approvals in place so that before we approvals before any remediation step can run.
+11. Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..) leveraging the No-code platform it provides.
```

Comment:
```suggestion

Once Aisera detects a major incident:

- It notifies the designated persona on the channel of their choice.
- Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the

Aisera can require approvals before a remediation task can run.

Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..), leveraging the no-code platform it provides.
```
Note that this sentence is still missing info at the end: "Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the".